src/source/Stun/Stun.c

```diff
@@ -541,68 +375,60 @@ STATUS deserializeStunPacket(PBYTE pStunBuffer, UINT32 bufferSize, PBYTE passwor
     PStunAttributeData pStunAttributeData;
     PStunAttributeChannelNumber pStunAttributeChannelNumber;
     BOOL fingerprintFound = FALSE, messaageIntegrityFound = FALSE;
-    PBYTE pData, pTransaction;
+    StunContext_t stunContext;
+    StunHeader_t stunHeader;
+    StunAttribute_t stunAttribute;
+    StunAttributeAddress_t stunMappedAddress;
+    UINT8 *pErrorPhrase, *pBuffer = NULL;
+    UINT16 errorPhraseLength, channelNumber, buffereLength, errorCode;
 
     CHK(pStunBuffer != NULL && ppStunPacket != NULL, STATUS_NULL_ARG);
     CHK(bufferSize >= STUN_HEADER_LEN, STATUS_INVALID_ARG);
 
-    if (!isBigEndian()) {
```

Comment:
Is the endianness handled in the STUN library?

Reply:
Yes it is handled in stun Library